core/domain/exp_services.py

```diff
@@ -976,16 +976,13 @@ def update_exploration_summary(exploration_id, contributor_id_to_add):
     Args:
         exploration_id: str. The id of the exploration whose summary is
             to be updated.
-        contributor_id_to_add: str or None. The user_id of user who have
+        contributor_id_to_add: str|None. The user_id of user who have
             contributed (humans who have made a positive (not just a revert)
             update to the exploration's content) will be added to the list of
-            contributours for the exploration if the argument is not None and it
+            contributors for the exploration if the argument is not None and it
             is not a system id.
     """
-    exploration = exp_fetchers.get_exploration_by_id(exploration_id)
-    exp_summary = compute_summary_of_exploration(
-        exploration, contributor_id_to_add)
-    save_exploration_summary(exp_summary)
+    create_exploration_summary(exploration_id, contributor_id_to_add)
```

Comment:
Just a thought -- for this and collections etc., if the functions are the same, maybe we should just combine these methods into a single method called regenerate_exp_summary()?